timemachine/observables/rmsd.py

```diff
@@ -0,0 +1,55 @@
+# This is ported from Matt Harrigan's TF code:
+# https://github.com/mdtraj/tftraj/blob/master/tftraj/rmsd.py
+
+import jax.scipy
+import jax.numpy as np
+
+
+def optimal_rotational_quaternion(r):
+    """Just need the largest eigenvalue of this to minimize RMSD over rotations
+    
+    References
+    ----------
+    [1] http://dx.doi.org/10.1002/jcc.20110
```

Comment:
It’s amazing to me how this gets recycled, although the gradient is a nice touch.

This is the earliest ref I have for these methods:

http://www.ccl.net/cca/software/SOURCES/C/quaternion-mol-fit/quatfit.txt.shtml

Reply:
Everyday i'm standin' on the shoulders of giants

Reply:
This was my goto for rmsd:  https://theobald.brandeis.edu/qcp/

Note that in the later papers one of the coauthors is now head of nx at novartis...

https://theobald.brandeis.edu/qcp/

Reply:
Nice! I ported that to the GPU back in 2012 during grad school:

https://onlinelibrary.wiley.com/doi/10.1002/jcc.23110